mantidproject/mantid
# Python 2 to 3 migration Algorithms Group 5

This fixes the algorithms & unit tests of group 5 in #16739.

To test:

The build servers will check the Python 2 unit tests so they should still work.To check for Python 3,
build the framework against Python 3 and run `ctest -R ^PythonInterface` - all of these tests should now pass.

Some of the algorithms are not covered by unit tests. System tests do not work under Python 3 yet, so these algorithms might be broken in Python 3.

Fixes #16937.

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
